### DISCUSSION TITLE
adding multiple tasks at the same time

### DISCUSSION THREAD

**Discussion by tomasdurham** [ransome1/sleek#376] [Ideas]
Hi, it would be awesome if it was possible to add multiple tasks as one list. The list could be written directly in the same box "Add new Task" e.g.:
1.task due:2022-02-15 +project1
2.task due:2022-02-15 +project2
3.task due:2022-02-15 +project3
...

**Comment by zerodat** [comment]
The great thing about todo.txt format is that you could open the todo list in a text editor to easily add several tasks all at once.  (At least until sleek has this feature. )

**Reply by tomasdurham** [reply]
Yes I now, I use sometimes Codium to add some multiple tasks.
Hope sleek will have this feature :)

**Comment by aisbergde** [comment]
using a text editor is also exactly, what I am doing to create a list of similar tasks. This is a major advantage, that I can use sleek to create new tasks, but in parallel I can use the text editor. And both together are working fine.

**Reply by tomasdurham** [reply]
that's right. But it is inconvenient to switch between different tools (text editors, Sleek ...) for different usage needs. I currently use mainly the app sleek. It would make the use more comfortable for me and probably others, if several tasks can be entered simultaneously in sleek.  nice to have :+1:

**Comment by ransome1** [comment]
> Hi, it would be awesome if it was possible to add multiple tasks as one list. The list could be written directly in the same box "Add new Task" e.g.: 1.task due:2022-02-15 +project1 2.task due:2022-02-15 +project2 3.task due:2022-02-15 +project3 ...

@tomasdurham @aisbergde sleek has now a setting for this. It is documented here: https://github.com/ransome1/sleek/wiki/Multi%E2%80%90line-text-field#bulk-todo-creation
